components/dashboards-web-component/src/designer/components/DashboardRenderer.jsx

```diff
@@ -102,12 +102,16 @@ export default class DashboardRenderer extends Component {
         }
         if (this.props.pageId === nextProps.pageId) {
             // Check whether the page height is difference for the same page.
-            let newHeight = nextProps.dashboard.pages.find(p => p.id === nextProps.pageId).height;
-            if (this.pageHeight !== newHeight) {
-                this.pageHeight = newHeight;
-                this.destroyGoldenLayout();
-                return true;
-            }
+            let changingPage = nextProps.dashboard.pages.find(p => p.id === nextProps.pageId);
```

Comment:
Not required to assign to a new variable as changingPage is not used elsewhere.

Reply:
```suggestion
            let changingPage = nextProps.dashboard.pages.find(p => p.id === nextProps.pageId);
            let newHeight = (changingPage === undefined) ? undefined:changingPage.height;
```

changingPage variable is used in the next line in two places to verify whether its undefined and for the assigning of height